contentcuration/contentcuration/frontend/shared/vuex/file/utils.js

```diff
@@ -61,6 +64,39 @@ export function storageUrl(checksum, file_format) {
   return `/content/storage/${checksum[0]}/${checksum[1]}/${checksum}.${file_format}`;
 }
 
+export async function getH5PMetadata(fileInput, metadata) {
```

Comment:
I think it would be a little cleaner if we didn't modify the `metadata` object here, and instead just defined `const metadata = {}`, then where this function is invoked, it could be assigned to the metadata object.

I don't think we need to fix this now, but it would be a purer function if it didn't cause a side effect to an input.